pages/admin/request/create-replacement.tsx

```diff
@@ -58,19 +72,120 @@ export default function CreateReplacement() {
     billingPostalCode: '',
   });
 
+  const toast = useToast();
+  const router = useRouter();
+
+  const [getApplicant] = useLazyQuery<GetApplicantResponse, GetApplicantRequest>(
+    GET_APPLICANT_QUERY,
+    {
+      onCompleted: ({ applicant: data }) => {
```

Comment:
```suggestion
      onCompleted: ({ applicant }) => {
```
Let's leave this object called `applicant` so that we have context below.